Composer/packages/lib/code-editor/src/hooks/useEditorToolbarPopExpandItem.tsx

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { createSvgIcon } from '@fluentui/react-icons';
+import { FluentTheme, NeutralColors } from '@uifabric/fluent-theme';
+import formatMessage from 'format-message';
+import { ICommandBarItemProps } from 'office-ui-fabric-react/lib/CommandBar';
+import React from 'react';
+
+import { defaultMenuHeight } from '../lg/constants';
+import { withTooltip } from '../utils/withTooltip';
+
+const itemButtonStyles = {
+  root: {
+    fontSize: FluentTheme.fonts.small.fontSize,
+    height: defaultMenuHeight,
+  },
+  menuIcon: { fontSize: 8, color: NeutralColors.black },
```

Comment:
should you use `FluentTheme` for the menu icon as well?

Reply:
yeah, why not, works out of the box once we try a dark theme 😄